#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_gcc-4.3.dpatch by LI Daobing <lidaobing@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix FTBFS in gcc-4.4
## DP: see http://sourceforge.net/support/tracker.php?aid=2485257

@DPATCH@
diff -urNad qterm~/src/main.cpp qterm/src/main.cpp
--- qterm~/src/main.cpp	2008-11-28 18:57:39.000000000 +0800
+++ qterm/src/main.cpp	2009-01-04 18:51:14.000000000 +0800
@@ -23,6 +23,8 @@
 #include <Python.h>
 #endif
 
+#include <stdio.h>
+
 #if !defined(_OS_WIN32_) && !defined(Q_OS_WIN32)
 #include <sys/stat.h>
 #include <errno.h>
diff -urNad qterm~/src/qtermbuffer.cpp qterm/src/qtermbuffer.cpp
--- qterm~/src/qtermbuffer.cpp	2009-01-04 18:49:48.000000000 +0800
+++ qterm/src/qtermbuffer.cpp	2009-01-04 18:50:21.000000000 +0800
@@ -18,7 +18,7 @@
 #include <QString>
 #include <QRect>
 #include <QRegExp>
-// #include <stdlib.h>
+#include <stdlib.h>
 // #include <stdio.h>
 // #include <math.h>
 namespace QTerm
diff -urNad qterm~/src/ssh/transport.cpp qterm/src/ssh/transport.cpp
--- qterm~/src/ssh/transport.cpp	2008-03-22 15:06:01.000000000 +0800
+++ qterm/src/ssh/transport.cpp	2009-01-04 18:50:48.000000000 +0800
@@ -12,6 +12,8 @@
 #include "transport.h"
 #include <QtCore/QString>
 
+#include <stdint.h>
+
 #ifdef SSH_DEBUG
 #include <QtDebug>
 extern void dumpData(const QByteArray & data);
